app/views/publishers/authentication_fallback_mailer/sign_in_fallback.text.erb

```diff
@@ -0,0 +1,7 @@
+# <%= t("app.title") %>
+
+<%= t("publishers.temp_login.email.heading") %>
```

Comment:
Missing the `##` second level heading here

Reply:
So before changing anything, the email looked like this:

![Screenshot 2022-01-10 at 12 01 01](https://user-images.githubusercontent.com/30624173/148762823-7ac0f368-73f8-456c-bb60-e60418981cb7.png)

I was playing around with this and it seemed both 

```
# <%= t("app.title") %>

## <%= t("publishers.temp_login.email.heading") %>

<%= t("publishers.temp_login.email.body") %>

<%= choose_organisation_link(@login_token) %>

```
and

```
# <%= t("app.title") %>

<%= t("publishers.temp_login.email.heading") %>

<%= t("publishers.temp_login.email.body") %>

<%= choose_organisation_link(@login_token) %>

```

look the same. Both seem to create p elements?

Reply:
With second level heading 

![Screenshot 2022-01-10 at 12 07 30](https://user-images.githubusercontent.com/30624173/148763685-3e440b95-7330-42d5-95fe-f4b37044207a.png)

Without:

![Screenshot 2022-01-10 at 12 08 13](https://user-images.githubusercontent.com/30624173/148763731-c8777303-4420-4d6c-9d3d-e5ee340e0ae2.png)

Reply:
Huh, I reckon it might be because Notify doesn't actually support second-level headings. In which case, carry on!